api/models/user.js

```diff
@@ -44,6 +44,24 @@ function toJSON() {
   });
 }
 
+const tableOptions = {
+  tableName: 'user',
+  classMethods: {
+    associate,
+  },
+  instanceMethods: {
+    toJSON,
+  },
+  paranoid: true,
+  scopes: {
```

Comment:
whoooooaaa! I did not know about `scopes` in Sequelize. That looks super useful!

Reply:
yeah! the only issue with them is that support for includes and nested includes is not great.